doc/mkrepos.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Generates a YAML file in the format expected by lsst-build --repos=<yaml>
+# from a directory populated git clones
+
+# oneliner to generate repo list from current lsstsw/lsst_build "build" dir
+repos=($(find ~lsstsw/build -maxdepth 2 -type d -name .git -exec grep url {}/config \; | cut -d' ' -f3))
+
+echo "---"
+for r in "${repos[@]}"; do
+  basename=${r##*/}
+  humanname=${basename%%.git}
+  clonedir=repos/${humanname}
+  url=$(echo $r | perl -pi -e 's|github.com/LSST|github.com/lsst|')
```

Comment:
To demonstrate that github does special handling to make the org name case insensitive.  It seems to issue a redirect for HTTP such that the remote in the clone is uppercased anyways.

We're planning to float a proposal at the hackathon to lowercase the github org name.

The `-i` was just my brain running on autopilot.  I'll remove it in a sec.